app/grandchallenge/archives/templates/archives/archive_cases_row.html

```diff
@@ -42,11 +43,14 @@
 {{ object.series_description }}
 <split></split>
 
-<a href="{% url 'workstations:workstation-session-create' slug=archive.workstation.slug %}?{% workstation_query image=object config=archive.workstation_config %}">
-    <span class="badge badge-primary">
-        <i class="fa fa-eye"></i> View Image
-    </span>
-</a>
+<button class="btn badge badge-primary"
+        data-session-control
+        data-create-session-url='{% url 'workstations:workstation-session-create' slug=archive.workstation.slug %}'
+        data-session-origin="https://us-east-1.{{ request.site.domain }}"
+        data-workstation-query='{% workstation_query image=object config=archive.workstation_config %}'
+        data-workstation-window-identifier='workstation-{{ object|meta_attr:'app_label' }}-{{ object|meta_attr:'model_name' }}'>
```

Comment:
Should these be `object` (image) or `archive`? Is `model_name` necessary?

Reply:
I would say `archive`